tests/foreman/destructive/test_clone.py

```diff
@@ -90,7 +90,10 @@ def test_positive_clone_backup(
     for repo in getattr(constants, f"OHSNAP_RHEL{rhel_version}_REPOS"):
         sat_ready_rhel.enable_repo(repo, force=True)
     # Enabling satellite module
-    assert sat_ready_rhel.execute(f'dnf module enable -y satellite:el{rhel_version}').status == 0
+    if str(settings.server.version.rhel_version)[0] != '9':
+        assert (
+            sat_ready_rhel.execute(f'dnf module enable -y satellite:el{rhel_version}').status == 0
```

Comment:
For this test, afaik satellite-clone anyway has dependancy of satellite package which will be anyway installed, so can we use the same helper here and install satellite with its dependancies before satellite-clone package, wdyt?